pkg/data/management/machinedriver_data.go

```diff
@@ -94,6 +96,9 @@ func addMachineDrivers(management *config.ManagementContext) error {
 	if err := addMachineDriver(ExoscaleDriver, "local://", "", "", []string{"api.exoscale.ch"}, false, true, management); err != nil {
 		return err
 	}
+	if err := addMachineDriver(GoogleDriver, "local://", "", "", []string{"www.googleapis.com"}, true, true, management); err != nil {
```

Comment:
we don't need to whitelist `local://` since we're not going call rancher for google driver (thanks @dramich)

Reply:
@cmurphy To expand on this the whitelist is URLs that rancher will proxy for the UI. The UI calls to providers through rancher to talk to the provider API. Depending on the provider it could be getting instance sizes, zones, etc. For Google, there will not be a UI portion that needs to talk to google so we don't want to whitelist a URL that rancher will not be using.

Reply:
Do you know if this whitelist is only for machine ? I believe the UI will need to make calls to gcp to get data for GKE, but that might be a different mechanism.

Reply:
Same whitelist technically. If the UI needs to do that then something needs to be whitelisted, but it should probably be `container.googleapis.com` based on https://cloud.google.com/kubernetes-engine/docs/reference/rest

Reply:
Machine is running arbitrary code and can call whatever it wants; the whitelist is only for the /meta/proxy the UI uses

Reply:
We are aiming at using the GKE specific meta proxy for now which does not need this whitelist, only the generic `meta/proxy` does. We can leave this for now as it shouldn't hurt anything and we may support GCP VMs in future.

Reply:
If we don't need it we should delete it. It's also not the correct URL for GCE.

Reply:
That works also. Is that not right URL though ? Minus the `www` part:  `compute.googleapis.com`

Reply:
It depends on which API is being used, I had been thinking this whitelist would be used to proxy auth requests and I had thought that `www.googleapis.com` would be used for that, but it might actually be `accounts.google.com` or `oauth2.googleapis.com`, I'm not sure. Removing it for now.